scripts/migrate-to-bcd.ts

```diff
@@ -0,0 +1,90 @@
+import assert from 'node:assert';
+import fs from 'node:fs';
+import path from 'node:path';
+import {fileURLToPath} from "node:url";
+import { fdir } from 'fdir';
+import { features } from '../index.js';
+
+export const BCD_PATH = process.env.BCD_PATH
+  ? path.resolve(process.env.BCD_PATH)
+  : fileURLToPath(new URL("../../browser-compat-data", import.meta.url));
+
+// Map from api.CoolThing.something to cool-thing
+const bcdToFeature = new Map();
+
+// Set of the first part of the BCD path encountered.
+const bcdDirs = new Set();
+
+for (const [feature, {compat_features}] of Object.entries(features)) {
+    if (!compat_features) {
+        continue;
+    }
+    for (const key of compat_features) {
+        assert(!bcdToFeature.has(key), `${key} is used in compat_features twice`);
```

Comment:
This is now enforced in [`index.ts`#L170-L180](https://github.com/web-platform-dx/web-features/blob/9eb029a6adc104588d4b9c45819722f064e5075b/index.ts#L170-L180), so we don't need to duplicate this check here.

```suggestion
```